examples/pytorch/deepergcn/models.py

```diff
@@ -0,0 +1,121 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import dgl.function as fn
+
+from ogb.graphproppred.mol_encoder import AtomEncoder
+from dgl.nn.pytorch.glob import SumPooling, AvgPooling, MaxPooling
+from modules import norm_layer
+from layers import GENConv
+
+
+class DeeperGCN(nn.Module):
+    r"""
+
+    Description
+    -----------
+    Introduced in `DeeperGCN: All You Need to Train Deeper GCNs <https://arxiv.org/abs/2006.07739>`_
+
+    Parameters
+    ----------
+    dataset: str
+        Name of ogb dataset.
+    node_feat_dim: int
+        Size of node feature dimension.
+    edge_feat_dim: int
+        Size of edge feature dimension.
+    hid_dim: int
+        Size of hidden dimension.
+    out_dim: int
+        Size of output dimension.
+    num_layers: int
+        Number of graph convolutional layers.
+    dropout: float
+        Dropout rate. Default is 0.
+    norm: str
+        Type of ('batch', 'layer', 'instance') norm layer in MLP layers. Default is 'batch'.
+    pooling: str
+        Type of ('sum', 'mean', 'max') pooling layer. Default is 'mean'.
+    beta: float
+        A continuous variable called an inverse temperature. Default is 1.0.
+    lean_beta: bool
+        Whether beta is a learnable weight. Default is False.
+    aggr: str
+        Type of aggregator scheme ('softmax', 'power'). Default is 'softmax'.
+    mlp_layers: int
+        Number of MLP layers in message normalization. Default is 1.
+    """
+    def __init__(self,
+                 dataset,
+                 node_feat_dim,
+                 edge_feat_dim,
+                 hid_dim,
+                 out_dim,
+                 num_layers,
+                 dropout=0.,
+                 norm='batch',
+                 pooling='mean',
+                 beta=1.0,
+                 learn_beta=False,
+                 aggr='softmax',
+                 mlp_layers=1):
```

Comment:
Guess we can remove `batch`, `pooling`. `aggr` is fine as people might be interested in checking the implementation.